src/main/java/eu/tornplayground/tornapi/TornApi.java

```diff
@@ -17,6 +19,7 @@ public class TornApi {
     private final ApiConnector connector;
     private final KeyProvider keyProvider;
     private String defaultComment;
+    private ResponseCache defaultResponseCache;
```

Comment:
I would rename this, and the first `withDefaultResponseCache` to not include default.